pkg/autodiscovery/listeners/docker.go

```diff
@@ -210,16 +221,23 @@ func (l *DockerListener) createService(cID string) {
 		}
 	}
 
+	checkNames, err := getCheckNamesFromLabels(cInspect.Config.Labels)
+	if err != nil {
+		log.Errorf("Error getting check names from docker labels on container %s: %v", cID, err)
```

Comment:
do we want to create the service if the function return an error?

In my optinion, if someone put the labels, but the payload is wrong. we should not create the service.

Reply:
the config validation logic is not in the service listener, it's handled by the provider and the autoconfig entity.